lib/create.rb

```diff
@@ -136,3 +135,48 @@ def self.create_from(authors)
     end
   end
 end
+
+class CreateGame < Create
+  require_relative './game'
+
+  def self.create
+    label = CreateLabel.create
+    genre = CreateGenre.create
+    author = CreateAuthor.create
+
+    print 'Multiplayer: Yes or No '
+    multiplayer = gets.chomp.strip
+    print 'Last played at: '
+    last_played_at = gets.chomp.strip
+    print 'Publish Date: '
+    publish_date = gets.chomp.strip
+
+    game = Game.new(multiplayer, last_played_at, publish_date)
+    game.label = label
+    game.genre = genre
+    game.author = author
+    game
```

Comment:
Would be better to keep on line above the return value to separate it from the logic.